kernel/src/utils.rs

```diff
@@ -1,13 +1,11 @@
 //! Various utility functions/macros used throughout the kernel
+use crate::{DeltaResult, Error};
+use delta_kernel_derive::internal_api;
```

Comment:
nit: revert to position below (usually do `std` imports, then 3p crates, then this crate)